domains/vulpine.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "VulpineFriend87",
+           "email": "",
+           "discord": "911268706112397392"
+        },
+    
+        "record": {
+            "A": ["69.30.249.53"]
```

Comment:
Please give a preview of your website hosted in DBH.

Reply:
http://pnode3.danbot.host:8503/